storage/csv.go

```diff
@@ -169,10 +173,29 @@ func (c *CSVBatch) PersistModel(ctx context.Context, m interface{}) error {
 		for i, f := range t.fields {
 			fv := value.FieldByName(f)
 			fk := fv.Kind()
-			if (fk == reflect.Slice || fk == reflect.Map || fk == reflect.Ptr || fk == reflect.Chan || fk == reflect.Func) && fv.IsNil() {
+			if (fk == reflect.Slice || fk == reflect.Map || fk == reflect.Ptr || fk == reflect.Chan || fk == reflect.Func || fk == reflect.Interface) && fv.IsNil() {
```

Comment:
nit: Worth breaking these into seaparate lines?

Reply:
I don't think it would add much to readability. Besides, we all have screen resolutions beyons 640x480 :grinning: